signer/main.go

```diff
@@ -11,17 +12,35 @@ import (
 
 	"github.com/aws/aws-sdk-go/aws/session"
 	v4 "github.com/aws/aws-sdk-go/aws/signer/v4"
+	"github.com/google/uuid"
 )
 
+// call with UID to generate a UID, or with
+// -expectedStatus=200 REQUEST <METHOD> <URL> <REQUEST BODY> to make a test request
```

Comment:
Not a blocker, but we may want to consider renaming this module since it's doing more than signing now